### PR TITLE
Improve view titles

### PR DESCRIPTION
**Pull request by alexweininger**
image

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         context.subscriptions.push(ext.rootAccountTreeItem);
         ext.appResourceTree = new AzExtTreeDataProvider(ext.rootAccountTreeItem, 'azureResourceGroups.loadMore');
         context.subscriptions.push(ext.appResourceTreeView = vscode.window.createTreeView('azureResourceGroups', { treeDataProvider: ext.appResourceTree, showCollapseAll: true, canSelectMany: true }));
+        ext.appResourceTreeView.description = 'Remote';
```
Comment (bwateratmsft) [inline src/extension.ts:55]: Should we localize?

**Review by bwateratmsft — APPROVED** [review]

File: src/extension.ts [hunk 2]
```diff
@@ -52,6 +53,7 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
         context.subscriptions.push(ext.rootAccountTreeItem);
         ext.appResourceTree = new AzExtTreeDataProvider(ext.rootAccountTreeItem, 'azureResourceGroups.loadMore');
         context.subscriptions.push(ext.appResourceTreeView = vscode.window.createTreeView('azureResourceGroups', { treeDataProvider: ext.appResourceTree, showCollapseAll: true, canSelectMany: true }));
+        ext.workspaceTreeView.description = localize('remote', 'Remote');
```
Comment (bwateratmsft) [inline src/extension.ts:56]:
```suggestion
        ext.appResourceTreeView.description = localize('remote', 'Remote');
```
Bad copypasta

Comment (alexweininger) [reply src/extension.ts:56]: Gooooood catch

Comment (bwateratmsft) [reply src/extension.ts]: image

**Pull request merged**